lib/mix/gear.new.ex

```diff
@@ -103,13 +103,22 @@ defmodule Mix.Tasks.Antikythera.Gear.New do
 
   defp gen_files(gear_name, dest_dir, instance_dep) do
     Mix.Generator.create_directory(dest_dir)
-    template_dir = Path.join(:code.priv_dir(:antikythera), "gear_template")
     binding = [gear_name: gear_name, gear_name_camel: Macro.camelize(gear_name), instance_dep: inspect(instance_dep)]
-    Enum.each(template_files(template_dir), fn template_path ->
-      dest_rel_path = template_path |> Path.relative_to(template_dir) |> String.replace("gear_name", gear_name)
-      dest_path = Path.join(dest_dir, dest_rel_path)
-      content = EEx.eval_file(template_path, binding)
-      Mix.Generator.create_file(dest_path, content)
+    instance_name = elem(instance_dep, 0)
+    dir1 = :code.priv_dir(:antikythera)
+    dirs = :code.priv_dir(instance_name)
+           |> case do
+             {:error, _} -> [dir1]
+             dir2        -> [dir1, dir2]
+           end
+    Enum.each(dirs, fn dir ->
+      template_dir = Path.join(dir, "gear_template")
+      Enum.each(template_files(template_dir), fn template_path ->
+        dest_rel_path = template_path |> Path.relative_to(template_dir) |> String.replace("gear_name", gear_name)
+        dest_path = Path.join(dest_dir, dest_rel_path)
+        content = EEx.eval_file(template_path, binding)
+        Mix.Generator.create_file(dest_path, content)
```

Comment:
Could you tell me which is your plan?

1. instance repository has only template files which are not included in this repository.
2. instance repository has all the template files for a gear project.

For the latter case, if there is a conflict between the instance repository and this repository, [create_file/3](https://hexdocs.pm/mix/Mix.Generator.html#create_file/3) asks for user confirmation.
Is this as intended?

Reply:
Thanks. It isn't my intention. I'll fix.

Reply:
fixed: 2ba123db5cfe940aa335353b8423faf96d5d443d